tests/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+include(CTest)
```

Comment:
`CTest` does not have to be included (`enable_testing()` should be enough). `include(CTest)` calls `enable_testing()` and creates a `BUILD_TESTING` option that is `ON` by default.

Reply:
Will do. Thanks for the tip!